data/sqlite3/db.go

```diff
@@ -12,7 +12,7 @@ func NewDB(env string) (*sqlx.DB, error) {
 	// https://github.com/mattn/go-sqlite3/issues/274#issuecomment-232942571
 	// enable a busy timeout for concurrent load. keep it short. the busy timeout can be harmful
 	// under sustained load, but helpful during short bursts.
-	return sqlx.Connect("sqlite3", fmt.Sprintf("./%v.db?cache=shared&_busy_timeout=200", env))
+	return sqlx.Connect("sqlite3", fmt.Sprintf("%v?cache=shared&_busy_timeout=200", env))
```

Comment:
Thanks for the patch!

My only hesitation here is backwards compatibility. What if we check to see whether the path contains a `.` and assume it's a newer style absolute path? Do you think that would work?

Reply:
Ok, i think that would be perfect